lewis_emulators/tpg300/device.py

```diff
@@ -55,6 +55,25 @@ class ReadState(Enum):
     FSB = "FSB"
     SPS = "SPS"
 
+    """ 
+    This object represents settings for a circuit in the device.
+        these settings are: high_threshold(float), high_exponent(int),
+        low_threshold(float), low_exponent(int), circuit_assignment(1|2|2|4|A|B)
+    """
```

Comment:
Style note; docstrings usually go inside a class or method, just after the class (or method) declaration itself.

Like this:

```python
class SomeClass:
    """
    Some documentation about the class
    """

    def some_method(self, args):
        """
        Docstring describing some_method
        """
        pass
```